sdcm/cluster.py

```diff
@@ -605,6 +608,52 @@ def cpu_cores(self):
         cores = int(cores) if cores else None
         return cores
 
+    @property
+    def scylla_shards(self):
+        return self.smp or self.cpuset
+
+    @property
+    def cpuset(self):
+        # Possible output of grep:
+        #   'CPUSET="--cpuset 1-7 "'
+        #   'CPUSET="--cpuset 1-7,9-15 "'
+        grep_result = self.remoter.run('grep "^CPUSET" /etc/scylla.d/cpuset.conf')
+        scylla_cpu_set = re.findall(r'(\d+)', grep_result.stdout)
+        self.log.debug(f"CPUSET on node {self.name}: {grep_result}")
+
+        if not scylla_cpu_set:
+            self.log.error(f'Failed to parse CPUSET from "/etc/scylla.d/cpuset.conf". '
+                           f'Unexpected CPUSET is "{grep_result}"')
+            cpuset = self.cpu_cores
+        else:
+            cpuset = int(scylla_cpu_set[1]) - int(scylla_cpu_set[0]) + 1
+
+            if len(scylla_cpu_set) == 4:
+                cpuset += int(scylla_cpu_set[3]) - int(scylla_cpu_set[2]) + 1
+
+        return cpuset
+
+    @property
+    def smp(self):
+        """
+        Example of SCYLLA_ARGS:
+        SCYLLA_ARGS="--blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc
+        --abort-on-internal-error 1 --abort-on-ebadf 1 --enable-sstable-key-validation 1 --smp 6 --log-to-syslog 1
+        --log-to-stdout 0 --default-log-level info --network-stack posix"
+        """
+        grep_result = self.remoter.run('grep "^SCYLLA_ARGS=" /etc/sysconfig/scylla-server')
```

Comment:
create propeprty `scylla_server_sysconfig_path`
use:
       `scylla_server_config = f"/etc/{'sysconfig' if self.distro.is_rhel_like else 'default'}/scylla-server"`
use it in `sdcm.cluster.BaseNode.config_setup`

Reply:
done